cpanfile

```diff
@@ -0,0 +1,10 @@
+# Force an up-to-date version so TidyAll dependencies can be installed and
```

Comment:
@perlpunk what do you think about adding this file? Feels a bit weird to have a manually maintained cpanfile when we already supply update-deps so maybe as an alternative we want to make update-deps work on this "template repo" itself?

Reply:
I think it's needed for the perltidy check.